### PR TITLE
v0.6.0 - created new components for additional icons

### PR DESCRIPTION
**Pull request by sonnyprince**
Addition of:
Cash, Clock, Local Legend, Map pin, Padlock and Stopwatch icons.

into f-vue-icons package.

### CODE REVIEW

File: packages/f-vue-icons/src/components/LocalLegendIcon/LocalLegendIcon.vue [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<template>
+    <local-legend-icon :class="[iconClasses]" />
```
Comment (dandel10n) [inline packages/f-vue-icons/src/components/LocalLegendIcon/LocalLegendIcon.vue]: There are 'c-icon c-icon--smth' classes for all other icons, are they not in use here?

Comment (sonnyprince) [reply packages/f-vue-icons/src/components/LocalLegendIcon/LocalLegendIcon.vue]: actually - we should be using `c-icon` - i'll update!

**Review by dandel10n — APPROVED** [review]

File: packages/f-vue-icons/src/components/MapPinIcon/MapPinIcon.vue [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<template>
+    <map-pin-icon :class="['c-icon c-icon--mappin', iconClasses]" />
```
Comment (DamianMullins) [inline packages/f-vue-icons/src/components/MapPinIcon/MapPinIcon.vue]: Should be `c-icon--mapPin`.

File: packages/f-vue-icons/src/components/index.js [hunk 3]
```diff
@@ -10,6 +10,12 @@ import OfferIcon from './OfferIcon';
 import PlusIcon from './PlusIcon';
 import SpicyIcon from './SpicyIcon';
 import VegetarianIcon from './VegetarianIcon';
+import PadlockIcon from './PadlockIcon';
```
Comment (DamianMullins) [inline packages/f-vue-icons/src/components/index.js]: This is mega nit-picky and OCD, but if you feel like it could you arrange these alphabetically?

Comment (sonnyprince) [reply packages/f-vue-icons/src/components/index.js]: done to all imports / exports 👍

**Review by DamianMullins — APPROVED** [review]

**Pull request merged**